README.md

```
@@ -13,6 +13,51 @@ This design system is built with:
+If you see a trust warning, run this once and retry:
+
+```sh
+mise trust
+```
+
+Run this only for repositories you trust.
+
+Add the activation command for your shell:
+
+zsh:
+```zsh
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)